crates/red_knot_vendored/vendor/typeshed/stdlib/VERSIONS

```diff
@@ -341,3 +341,4 @@ zipfile._path: 3.12-
 zipimport: 3.0-
 zlib: 3.0-
 zoneinfo: 3.9-
+red_knot: 3.0-
```

Comment:
We would obviously *not* put this in our vendored copy of `typeshed`. This was just the easiest option for this PoC.